ecosystem/cli/website.py

```diff
@@ -62,7 +62,10 @@ def _load_from_file(
     )
 
     # Label descriptions
-    label_descriptions = json.loads((resources_dir / "labels.json").read_text())
+    label_descriptions = {}
+    for _category in json.loads((resources_dir / "labels.json").read_text()).values():
+        for label in _category:
```

Comment:
You use the variable inside the loop

```suggestion
    for category in json.loads((resources_dir / "labels.json").read_text()).values():
        for label in category:
```